.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Did you mean to commit this change?

Reply:
my bad 😆 will merge this in after the VSCode PR so the diff is fixed